client/state/selectors/test/get-past-billing-transaction-error.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+/**
+ * Internal dependencies
+ */
+import { getPastBillingTransactionError } from 'state/selectors';
+
+describe( 'getPastBillingTransactionError()', () => {
+	const state = {
+		billingTransactions: {
+			individualTransactions: {
+				errors: {
+					123: false,
```

Comment:
As mentioned above, it would probably make more sense if we use error objects here, rather than `bool` values. This would make the tests more self-explicable and would document the code a bit better.